libcrux-ml-kem/src/mlkem512.rs

```diff
@@ -677,6 +677,34 @@ pub mod incremental {
         RANK_512 * 16 * 32
     }
 
+    /// The size of the key pair in bytes.
+    pub const fn key_pair_len() -> usize {
```

Comment:
I'm 99% of the way there on using this, but there's still one thing I think should be made public here: the expected lengths of CT1 and CT2.  I need to set up a receiver with the expected number of bytes I expect to receive:

```
  let receiver = ExpectToReceive(CT1_SIZE);
  while !receiver.done() {
    receiver.read_byte(b);
  }
```

Right now, I'm not seeing a good way to do this with the constnats and functions that are currently publicly exposed.  Of course, I could recompute the constants myself, but that seems a bit brittle.